charts/external-nodes/values.yaml

```diff
@@ -0,0 +1,29 @@
+nodeRegistarJob:
+  dockerRegistry:
+  image: codefresh/cli
+  imageTag: latest
+
+cf-api-token: 
+
+consul: http://cf-consul:8500
+
+## Set runtime environments
+runtimeEnvironments:
+  - name: default-nodes
+    extends: system/default
+    cluster: codefresh
+    protocol: http
+    #alternateLoggerConf: external-nodes
+    nodes:
+    - address: 
+      # name: node1
+      # port: 2376
+    # - address:
+      # name:
+      # port:
```

Comment:
if not used for documentation, please remove

Reply:
it is example of config for understanfing